src/scripts/achievements/AchievementTypes.ts

```diff
@@ -0,0 +1,17 @@
+enum achievementType {
```

Comment:
All of our enums so far have begun with a capital letter, it would be nice to keep that consistent

Reply:
ok